### DISCUSSION TITLE
OBO wide general cc-zero for labels, mappings and identifiers?

### DISCUSSION THREAD

**Discussion by matentzn** [OBOFoundry/OBOFoundry.github.io#2355] [General]
After various conversations with people from Wikidata and Industry, I am more convinced (55% convinced) now that we should steer the community towards CC-0 (public domain) "licensing". I think reducing barriers to uptake is now more important than concerns of "poaching" or "theft" - although I do see these as problematic as well, especially if a commercial entity simply copies our curated ontologies without attribution. 

Having any kind of license creates a barrier to reuse, and it makes it impossible to migrate terms into other spaces like Wikidata for example (or redistribute them in other cc0 content). I would like to try the following: 

Create a universal CC-Zero assumption for all OBO-hosted 
- ontology identifiers and 
- labels (rdfs:label) and
- xrefs and skos mappings
 
regardless of what other license restrictions the ontology otherwise imposes. Other contents (definitions, axioms, synonyms, etc) remain under the normal umbrella of the ontologies license.

I am wondering if I am a bit naive about the consequences this has for bad-faith actors (poachers). But I would like to believe that especially encouraging small enterprises and public KG projects to pick up OBO ontologies is worth this concession.

What do you think? If you agree, this could be realised in two ways:

1. We try to lobby all members of OBO and get buy-in from everyone
2. We could create a contract we host on the OBO Foundry website we have all participating ontologies "sign".

**Comment by balhoff** [comment]
A relevant issue that's been open for a while: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/285

**Comment by nataled** [comment]
I have no objection, but raise two points:
- I suspect there's a potential for confusing users if an ontology of interest is *not* CC0.
- For some ontologies that are CC-BY, the attribution stipulation is as simple as using the identifier. Does that make the identifiers effectively CC0?

**Comment by jamesaoverton** [comment]
We discussed this on the OFOC call 2023-05-16. There was some support, and also some concerns raised and some confusion.

1. Technically, are we talking about ODK generating a new artifact that we publish somewhere, and slap the CC0 license on? I worry that we have an increasing number of artifacts...
2. A concern was raised that if labels/IDs/mappings are CC0 but the ontology is not, users may be confused.

This would be easier if all OBO projects were just CC0. We discussed that on the call. Some people were in favour of pushing for CC0 while others think the best we can do is recommend it.

**Comment by hoganwr** [comment]
One interesting side effect, although it probably doesn't matter much
practically, is that if someone switched their ontology's license, then all
the old releases are still the original license (CC-BY in this scenario)
and the new releases are the new license going forward (CC0).  It would
only matter if there were a use case for using an old version either alone
or in combo with a new version (e.g. some derivative product that shows
evolution of definitions).
